docs/reference/manifest/versionoverrides.md

```diff
@@ -33,13 +33,14 @@ The following lists the required value of the **xmlns** value depending on the *
-|  **Description**    |  No   |  Describes the add-in. This overrides the `Description` element in any parent portion of the manifest. The text of the description is contained in a child element of the **LongString** element contained in the [Resources](./resources.md) element. The `resid` attribute of the **Description** element is set to the value of the `id` attribute of the `String` element that contains the text.|
+|  **Description**    |  No   |  Describes the add-in. This overrides the `Description` element in any parent portion of the manifest. The text of the description is contained in a child element of the **LongString** element contained in the [Resources](resources.md) element. The `resid` attribute of the **Description** element is set to the value of the `id` attribute of the `String` element that contains the text.|
 |  **Requirements**  |  No   |  Specifies the minimum requirement set and version of Office.js that the add-in requires. This overrides the  `Requirements` element in the parent portion of the manifest.|
-|  [Hosts](./hosts.md)                |  Yes  |  Specifies a collection of Office hosts. The child  Hosts element overrides the Hosts element in the parent portion of the manifest.  |
-|  [Resources](./resources.md)    |  Yes  | Defines a collection of resources (strings, URLs, and images) that other manifest elements reference.|
-|  [EquivalentAddins](./equivalentaddins.md)    |  No  | Specifies the native (COM/XLL) add-ins that are equivalent to the web add-in. The web add-in is not activated if an equivalent native add-in is installed.|
+|  [Hosts](hosts.md)                |  Yes  |  Specifies a collection of Office hosts. The child  Hosts element overrides the Hosts element in the parent portion of the manifest.  |
+|  [Resources](resources.md)    |  Yes  | Defines a collection of resources (strings, URLs, and images) that other manifest elements reference.|
+|  [EquivalentAddins](equivalentaddins.md)    |  No  | Specifies the native (COM/XLL) add-ins that are equivalent to the web add-in. The web add-in is not activated if an equivalent native add-in is installed.|
 |  **VersionOverrides**    |  No  | Defines add-in commands under a newer schema version. See [Implementing multiple versions](#implementing-multiple-versions) for details. |
-|  [WebApplicationInfo](./webapplicationinfo.md)    |  No  | Specifies details about the add-in's registration with secure token issuers, such as Azure Active Directory V2.0. |
+|  [WebApplicationInfo](webapplicationinfo.md)    |  No  | Specifies details about the add-in's registration with secure token issuers, such as Azure Active Directory V2.0. |
+|  [ExtendedPermissions](extendedpermissions.md) |  No  |  Specifies a collection of extended properties.<br><br>**Important**: Because the append on send feature is currently in preview, add-ins that use the `ExtendedPermission` element can't be published to AppSource or deployed via centralized deployment. |
```

Comment:
Will the important section call out every feature in the future that needs this element and is in preview that if it's in preview it won't be published? Why not make it generic saying that if a feature needs this element and its in preview then you wouldn't be able to publish to store...?

Reply:
No, this is only because it's the first feature. For subsequent features, we'll likely note this in the ExtendedPermission list of available values. For example, if PreviewFeature2 has ExtendedPermission value PreviewFeatureValue2, PreviewFeatureValue2 will be tagged as being in preview and add-ins that use this value can't/shouldn't be published to AppSource/CD.